### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by jeff-matthews**
- #5202 
- #5215 
- #5217 
- #5224 
- #5226 
- #5227

### CODE REVIEW

File: guides/v2.3/extension-dev-guide/declarative-schema/index.md [hunk 1]
```diff
@@ -28,8 +28,8 @@ Once a module is converted to the declarative schema approach, it cannot be reve
 
 * *Data patch* - A class that contains data modification instructions. It can have dependencies on other data or schema patches.
 
-* *Revertable data patch* - A patch that can be reverted as a module or patch is uninstalled or deleted. Revertable operations are Data Query Language (DQL) and Data Manipulation Language (DML) operations: INSERT, UPDATE.
-
+* *Revertable data patch* - A `data patch` that contains a `revert()` method (see [Reverting data patches]({{ page.baseurl }}/extension-dev-guide/declarative-schema/data-patches.html/#revertingDataPatches)) with provided operations that revert revertable changes caused by the module. Revertable operations are usually related to changes in the database.
```
Comment (erikmarr) [inline guides/v2.3/extension-dev-guide/declarative-schema/index.md]: Does the link actually work with the `/` before the anchor?

```suggestion
* *Revertable data patch* - A `data patch` that contains a `revert()` method (see [Reverting data patches]({{ page.baseurl }}/extension-dev-guide/declarative-schema/data-patches.html#revertingDataPatches)) with provided operations that revert revertable changes caused by the module. Revertable operations are usually related to changes in the database.
```

**Comment by jeff-matthews** [comment]
running tests

File: guides/v2.2/config-guide/bootstrap/magento-modes.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ You can run Magento in any of the following *modes*:
 				<li>Enables enhanced debugging</li>
 				<li>Shows custom <code>X-Magento-*</code> HTTP request and response headers</li>
 				<li>Results in the slowest performance</li>
-				<li>Does not show errors on the frontend</li></ul>
+				<li>Show errors on the frontend</li></ul>
```
Comment (erikmarr) [inline guides/v2.2/config-guide/bootstrap/magento-modes.md]:
```suggestion
				<li>Shows errors on the frontend</li></ul>
```

File: guides/v2.3/extension-dev-guide/declarative-schema/index.md [hunk 3]
```diff
@@ -38,3 +38,5 @@ Once a module is converted to the declarative schema approach, it cannot be reve
   * Performing data migration with inside DDL operations
   * Renaming tables, columns, and other entities
   * Adding partitions and options to a table
+
+* *Revertable schema patch* - A `schema patch` that contains a `revert()` method with a provided revert functionality. Like in the `Revertable data patch`, a revert functionality of the `schema patch` affects changes in the database only.
```
Comment (erikmarr) [inline guides/v2.3/extension-dev-guide/declarative-schema/index.md]:
```suggestion
* *Revertable schema patch* - A `schema patch` that contains a `revert()` method with the provided revert functionality. Like in the `Revertable data patch`, the revert functionality of the `schema patch` affects changes in the database only.
```

File: guides/v2.3/extension-dev-guide/declarative-schema/index.md (hunk 1 above)
Comment (jeff-matthews) [reply guides/v2.3/extension-dev-guide/declarative-schema/index.md]: Tests passed, so I think the answer is yes.

**Comment by jeff-matthews** [comment]
running tests

File: guides/v2.3/extension-dev-guide/declarative-schema/index.md (hunk 1 above)
Comment (jeff-matthews) [reply guides/v2.3/extension-dev-guide/declarative-schema/index.md]: Just confirmed that this works without the slash. It's already that way on production: https://devdocs.magento.com/guides/v2.3/extension-dev-guide/declarative-schema/data-patches.html#reverting-data-patches

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**